elements/lisk-transactions/README.md

```diff
@@ -2,6 +2,92 @@
 
 @liskhq/lisk-transactions is containing everything related to transactions according to the Lisk protocol
 
+## Introduction
+
+Transactions are the essential part of the blockchain applications created using Lisk SDK.
+
+The Lisk SDK provides a class [BaseTransaction](https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-transactions/src/base_transaction.ts) from which developers can inherit and extend from, to create **custom transaction types**.```
+The application-specific business logic for custom transaction types is defined according to an abstract [interface](#interface) that is common across all transaction types.
+
+All of the default transaction types of the Lisk SDK transactions implement the abstract interface of the base transaction, and therefore can be used as a role model for custom transactions.
+It's also possible to inherit from one of the default transaction types, in order to extent or modify them.
+
+The default transaction types each implement a different use-case of the Lisk network, i.e:
+
+0. Balance transfer (type 0),
+1. Second signature registration (type 1)
+1. Delegate registration (type 2)
+1. Delegate vote (type 3)
+1. Multisignature account registration (type 4)
+
+> The first 10 transaction types are reserved for the [Lisk protocol](https://lisk.io/documentation/lisk-protocol), don't use them to register custom transactions.
+
+For a complete list of all default transaction types, check out the section [Lisk Transactions](https://lisk.io/documentation/lisk-protocol/transactions) of the Lisk Protocol.
+
+Check out the Lisk SDK [Example Apps](https://github.com/LiskHQ/lisk-sdk-test-app) for simple [code examples of custom transaction types](https://github.com/LiskHQ/lisk-sdk-test-app/blob/development/hello_world/hello_transaction.js).
+
+### Lifecycle
+
+The lifecycle of a transaction in general in Lisk SDK can be summarized as follows:
+
+1. A transaction is created and signed (off-chain). The script to do it is in `src/create_and_sign.ts`.
+2. The transaction is sent to a network. This can be done by a third party tool (like `curl` or `Postman`), but also using Lisk Commander.
```

Comment:
```suggestion
2. The transaction is sent to a network. This can be done by a third party tool (like `curl` or `Postman`), but also using Lisk Commander (`lisk transaction:broadcast`).
```